app/services/wallets/balance/increase_service.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Wallets
+  module Balance
+    class IncreaseService < BaseService
+      def initialize(wallet:, credits_amount:)
+        super(nil)
+
+        @wallet = wallet
+        @credits_amount = credits_amount
+      end
+
+      def call
+        amount = BigDecimal(wallet.rate_amount) * credits_amount
+
+        wallet.update!(
+          balance: BigDecimal(wallet.balance) + amount,
+          credits_balance: BigDecimal(wallet.credits_balance) + credits_amount,
+          last_balance_sync_at: Time.zone.now,
```

Comment:
```suggestion
          last_balance_sync_at: Time.current,
```

Reply:
Fixed (and removed BigDecimal casting once my schema is fixed)